kobo/apps/stripe/views.py

```diff
@@ -11,3 +24,63 @@ class ProductViewSet(viewsets.ReadOnlyModelViewSet):
         livemode=settings.STRIPE_LIVE_MODE,
     ).prefetch_related('plan_set')
     serializer_class = ProductSerializer
+
+
+class SubscriptionViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    model = Subscription
+    serializer_class = SubscriptionSerializer
+    lookup_field = 'id'
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        user_orgs = self.request.user.organizations_organization.values_list(
+            'id',
+            flat=True,
+        ).all()
+        queryset = Subscription.objects.filter(
+            customer__subscriber_id__in=user_orgs,
+        )
+        return queryset
+
+    def list(self, request, *args, **kwargs):
```

Comment:
We do we need these custom list, retrieve functions? Is there a way we could use the default implementation instead?